client/src/GridImages.tsx

```diff
@@ -0,0 +1,150 @@
+    },
+    missionText: {
+        gridColumn: '5 / span 3',
+        textAlign: 'left',
+        fontSize: '1.4rem',
+        '&$row1': {
+            gridRow: '1 / span 1',
+        },
+        '&$row2': {
+            gridRow: '2 / span 1',
+        },
+        '&$row3': {
+            gridRow: '3 / span 1',
+        },
+        '&$row4': {
+            gridRow: '4 / span 1',
+        },
+    },
+    row1: {
+    },
+    row2: {
+    },
+    row3: {
+    },
+    row4: {
+    },
+    missionTitle: {
+        fontWeight: 'bold',
+        fontSize: '1.4rem',
+    },
+    '@media screen and (max-width: 1100px)': {
+        missionContent: {
+            gridTemplateColumns: 'repeat(6, 1fr)',
+        },
+        missionText: {
+            gridColumn: '4 / span 3',
+        },
+    },
+    '@media screen and (max-width: 820px)': {
+        missionContent: {
+            gridTemplateColumns: 'repeat(4, 1fr)',
+            columnGap: '15px',
+            rowGap: '15px',
+        },
+        missionText: {
+            gridColumn: '1 / span 4',
+            '&$row1': {
+                gridRow: 'auto / span 1',
+            },
+            '&$row2': {
+                gridRow: 'auto / span 1',
+            },
+            '&$row3': {
+                gridRow: 'auto / span 1',
+            },
+            '&$row4': {
+                gridRow: 'auto / span 1',
+            },
+        },
+        missionWideImage: {
+            // gridRow: 'auto / span 1',
+        },
+    },
+    '@media screen and (max-width: 520px)': {
+        missionContent: {
+            gridTemplateColumns: 'repeat(3, 1fr)',
+        },
+        missionText: {
+            gridColumn: '1 / span 3',
+        },
+    },
+}));
+
+// type Props = {
+//     missionStatements: [],
+// };
+  
+function GridImages() {
+    
+    const classes = useStyles();
+//     // const { missionStatements } = props;
+
+    const smallImgs = ['sm', 'sm', 'sm', 'sm', 'sm', 'sm', 'sm', 'sm', 'sm'];
+
+    // This needs to be moved into About Us and passed in as props
+    let missionStatements = [
+        {row: 1, title: 'Mission Statement', text: loremIpsum.slice(0,97)},
+        {row: 2, title: 'Vision Statement', text: loremIpsum.slice(0,97)},
+        {row: 3, title: 'Values', text: loremIpsum.slice(0,97)},
+    ];
+
+    let rowClass = classes.row1;
+
+    return (
+        <Box className={`${classes.missionContent}`}>
+            <Box className={`${classes.missionWideImage}`}></Box>
+            {smallImgs.map((value, index) => {
+                return <Box className={`${classes.missionSmallImage}`}></Box>
+            })}
+            {missionStatements.map((statementItem, index) => {
+                if (statementItem.row === 1) rowClass = classes.row1;
+                if (statementItem.row === 2) rowClass = classes.row2;
+                if (statementItem.row === 3) rowClass = classes.row3;
+                if (statementItem.row === 4) rowClass = classes.row4;
```

Comment:
do you have a plan for this?
i think we can remove this, the `rowClass` var and the `classes.rowN` properties for now